db.h

```diff
@@ -12,11 +12,11 @@
 
 #define MIN_RETRY 1000
 
-#define REQUIRE_VERSION 70215
+#define REQUIRE_VERSION 70214
 
 static inline int GetRequireHeight(const bool testnet = fTestNet)
 {
-    return testnet ? 40000 : 1080000;
+    return testnet ? 0 : 25000;
```

Comment:
set the current block num for finding up to date seeders

Reply:
ie in the place of 25k blocks put the current block num

Reply:
The seeder can't find out the current block number, without hooking into a locally running daemon which it currently doesn't need to.  I set it to 25,000 but can set it higher for sure.

Reply:
updated to 72698